src/components/sections/dataset/dataSet-settings.js

```diff
@@ -277,17 +271,18 @@ class DataSetSettings extends React.Component {
                 disableSave: false,
             })
         },
-        onInputChange: e => {
-            e.preventDefault()
+        onInputChange: (e, key) => {
+            let name, value
+
+            typeof key === 'string'
+                ? ((name = key), (value = e.selected))
+                : ((name = e.name), (value = e.value))
```

Comment:
This is indirect logic, shouldn't use a ternary in this way if you're not assigning the result of the ternary to some variable.

There are two simpler, easier-to-read options:

```
const name = typeof key === 'string' ? key : e.name
const value = typeof key === 'string' ? e.selected : e.value
```

OR (less desirable because it doesn't allow the variables to be const, but a little cleaner because you don't need any ternary):

```
let name, value

if (key === 'string') {
  name = key
  value = e.selected
} else {
  name = e.name
  value = e.value
}
```